### PR TITLE
chore: update composer.scripts

### PR DESCRIPTION
**Pull request by kenjis**

### CODE REVIEW

**Comment by kenjis** [comment]
Can someone review?

File: composer.json [hunk 1]
```diff
@@ -59,14 +60,14 @@
             "Composer\\Config::disableProcessTimeout",
             "@cs",
             "@deduplicate",
-            "@analyze",
             "@inspect",
+            "@analyze",
```
Comment (paulbalandan) [inline composer.json:64]: Why this change?

Comment (kenjis) [reply composer.json:64]: The same as devkit:
https://github.com/codeigniter4/devkit/blob/d4ac84b92ede7a7eb5dc21b6aa0af7b4001efd5d/composer.json#L59-L66

**Review by paulbalandan — APPROVED** [review]
